### DISCUSSION TITLE
How to support new cc2652 USB adapter?

### DISCUSSION THREAD

**Discussion by shikon** [Koenkk/zigbee2mqtt#10525] [Devices]
After my old cc2531 died, because the flash got corrupted. 
I wanted to use the new cc2652 chip, however due to chip shortages the usb sticks listed on the website were sold out.
Therefore I designed my own and would like to start selling them also because I bought a batch of the chips. 
Any way to make sure my adapter is supported correctly and I don't give out USB sticks that are hard to support or thinker with?

It's based on the E72 ebyte chip, so I could easily flash the already available firmware but I would like to make sure that I can call it quality work...

**Comment by Koenkk** [comment]
Only thing I can think of is the LED pins, use the same as the others: https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin

**Reply by shikon** [reply]
Yeah I use the same pins, unfortunately only 1 LED pin is used in the E72 modules. 
So far so good, time to do the next run of PCB's and create a tutorial how to flash but it is basically the same as others...

Thanks a lot @Koenkk, great software and enabling a lot of freedom.
[edit]
Oh also should/can I send you one to test?
